infobip-openapi-mcp-core/src/main/java/com/infobip/openapi/mcp/auth/ScopeProperties.java

```diff
@@ -27,6 +27,10 @@ public record ScopeProperties(
      * Constructor with defaults for optional properties.
      */
     public ScopeProperties {
+        if (!enabled) {
+            enabled = true;
```

Comment:
We'll need to update javadoc on this class and table in README.md to document the new default.
Should probably look into ways to generate readme from javadoc so that we don't forget this kind of updates. 🤔  But that's for another task.